static/js/project_create.js

```diff
@@ -0,0 +1,45 @@
+const addForm = (name, event) => {
+  const formIndex = $(`#id_${name}-TOTAL_FORMS`).val();
+  const newForm = $(`#${name}-forms .template`)
+    .html()
+    .replace(/__prefix__/g, formIndex);
```

Comment:
What is this `__prefix__` business? Comment would help me here!

Reply:
Commented each new variable, hopefully makes sense now!